### PR TITLE
Night mode tweak for pre

### PR DESCRIPTION
**Pull request by gavin19**

### CODE REVIEW

**Comment by allthefoxes** [comment]
Fixes #2054

File: lib/modules/nightmode.css [hunk 1]
```diff
@@ -674,6 +674,7 @@
 	background-color: #345;
 }
 .res-nightmode .md code,
+.res-nightmode .md pre
 .res-nightmode .content .md pre {
```
Comment (andytuba) [inline lib/modules/nightmode.css]: did you want to delete this line, too?

File: lib/modules/nightmode.css [hunk 2]
```diff
@@ -674,6 +674,7 @@
 	background-color: #345;
 }
 .res-nightmode .md code,
+.res-nightmode .md pre
```
Comment (andytuba) [inline lib/modules/nightmode.css]: missing comma

**Comment by andytuba** [comment]
thanks!

**Comment by andytuba** [comment]
merged

**Pull request closed without merging**